src/types/component.rs

```diff
@@ -137,21 +167,41 @@ pub struct Text {
     pub id: Option<ComponentId>,
     pub text: Arc<str>,
 
+    /// Width of a texture that text will be rendered on. If not provided the resulting texture
+    /// will be sized based on the defined text, but limited to `max_width` value.
     pub width: Option<f32>,
+    /// Height of a texture that text will be rendered on. If not provided the resulting texture
+    /// will be sized based on the defined text, but limited to `max_width` value.
+    ///
+    /// It's an error to provide `height` if width is not defined.
     pub height: Option<f32>,
+    /// (default=7682) Width of a texture that text will be rendered on. Value is ignored if width
+    /// is defined.
     pub max_width: Option<f32>,
+    /// (default=4320) Height of a texture that text will be rendered on. Value is ignored if
+    /// height is defined.
     pub max_height: Option<f32>,
 
+    /// Font size in pixels.
     pub font_size: f32,
+    /// Distance between lines. Defaults to value of a `font_size` property.
     pub line_height: Option<f32>,
+    /// (default="#FFFFFFFF") Font color in `#RRGGBBAA` format.
     pub color_rgba: Option<RGBAColor>,
+    /// (default="#00000000") Background color in `#RRGGBBAA` format.
     pub background_color_rgba: Option<RGBAColor>,
+    /// (default="Verdana") Font family.
+    ///
+    /// Provide family-name for specific font. "generic-family" values like e.g. "sans-serif" will not work.
     /// https://www.w3.org/TR/2018/REC-css-fonts-3-20180920/#family-name-value   
-    /// use font family name, not generic family name
-    pub font_family: Option<String>, // TODO: Arc<str>
+    pub font_family: Option<Arc<str>>,
+    /// (default="normal") Font style. Font that you selected needs to support the selected weight.
```

Comment:
I would keep it consistent across the whole documentation. I prefer a non-personal form rather than addressing users like this.

```suggestion
    /// (default="normal") Font style. The selected font needs to support the selected weight.
```